CMakeLists.txt

```diff
@@ -195,6 +195,86 @@ else()
     set(OBAKE_SRC_FILES "${CMAKE_CURRENT_SOURCE_DIR}/src/boost_stack_trace.cpp" "${OBAKE_SRC_FILES}")
 endif()
 
+
+# Make obake header files accessible in Visual Studio IDE.
+if(YACMA_COMPILER_IS_MSVC)
+	set(OBAKE_HEADER_FILES
+	"${CMAKE_CURRENT_LIST_DIR}/include/obake/byte_size.hpp"
```

Comment:
@7ofNine any reason why you are using ``CMAKE_CURRENT_LIST_DIR`` instead of ``CMAKE_CURRENT_SOURCE_DIR``? I'd rather stick with the latter, if it works for you.

Reply:
I recently actually read a book about cmake and using cmake_current_list_dir is the revommended version. Seemingly it has to do with the scope of it. Here I don't think it makes any difference. I'll change it back to CMAKE_CURRENT_SOURCE_DIR.

Reply:
Ok cheers! It was mostly a matter of consistency with the rest of the build system, we can switch everything to ``CMAKE_CURRENT_LIST_DIR`` down the line.